src/containers/header/index.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import CButton from 'src/components/CButton';
+import CLink from 'src/components/CLink';
+
+import LogoIcon from 'src/svgs/LogoIcon';
+import LogoTypography from 'src/svgs/LogoTypography';
```

Comment:
Add Icon at the end of all imports of the icons or remove the logo from the beginning of all. make all of them similar